book: fix english (#3487)
kanidm/kanidm: book/src/support.md

### Python module

> The Python module will typically trail changes in functionality of the core Rust code, and will be

Reviewer: ```suggestion
The Python module will typically trail changes in functionality of the core Rust code, and has been
```

> developed only for our own needs - please feel free to add functionality or improvements, or

Reviewer: ```suggestion
developed as we have needed it - please feel free to add functionality or improvements, or
```
